azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/JUnitScenarioRunner.kt

```diff
@@ -199,6 +200,16 @@ class JUnitScenarioRunner<
                         } else {
                             testClass.primaryConstructor!!.call(*parameters)
                         }
+                        val sinceAnnotation = build.getAnnotation(Since::class.java)
+                        val scenarioVersion =
+                            sinceAnnotation?.implementationVersion?.filter { it.name == implementationInstance.implementationName }
+                                ?.singleOrNull()
+                        if (implementationInstance.supportsScenarioVersion(scenario, scenarioVersion) == false) {
```

Comment:
⛏️ I know you're keen on always using positive conditionals, but I find this harder to read than `if (!implementationInstance.supportsScenarioVersion(scenario, scenarioVersion))`.

Reply:
But that doesn't mean the same thing!

Reply:
Ah, I missed that `implementationInstance.supportsScenarioVersion(scenario, scenarioVersion)` is nullable.